docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Quick question, when is this file used?

Reply:
This is a mistake, I will remove that file